src/main/webapp/app/quiz/shared/questions/drag-and-drop-question/drag-and-drop-question.component.ts

```diff
@@ -72,6 +72,7 @@ export class DragAndDropQuestionComponent implements OnChanges, OnInit {
     @Input()
     set question(question) {
         this._question = question;
+        this.showingSampleSolution = false;
```

Comment:
I think we can reuse the method here instead

```suggestion
        this.hideSampleSolution();
```